api/src/services/bctw-service.ts

```diff
@@ -0,0 +1,194 @@
+import axios, { AxiosResponse } from 'axios';
+import { URLSearchParams } from 'url';
+import { ApiError, ApiErrorType } from '../errors/api-error';
+import { KeycloakService } from './keycloak-service';
+
+export interface IDeployDevice {
+  device_id: number;
+  frequency: number;
+  manufacturer: string;
+  model: string;
+  attachment_start: string;
+  attachment_end: string;
+  critter_id: string;
+}
+
+export interface IDeploymentUpdate {
+  deployment_id: string;
+  attachment_start: string;
+  attachment_end: string;
+}
+
+export interface IDeploymentRecord {
```

Comment:
Might a good idea to future proof these by turning them into Zod interfaces. For examples, search the codebase for `z.object({` and `z.infer`